test/api/v3/unit/middlewares/cronMiddleware.js

```diff
@@ -107,53 +115,57 @@ describe('cron middleware', () => {
     await task.save();
     await user.save();
 
-    cronMiddleware(req, res, (err) => {
+    cronMiddleware(req, res, () => {
       Tasks.Task.findOne({_id: task}, function (secondErr, taskFound) {
         expect(secondErr).to.not.exist;
         expect(taskFound).to.not.exist;
-        done(err);
       });
     });
   });
 
-  it('should call next if user was not modified after cron', async (done) => {
+  it('should call next if user was not modified after cron', async () => {
+    execStub.returns(Bluebird.resolve(null));
+
     let hpBefore = user.stats.hp;
     user.lastCron = moment(new Date()).subtract({days: 2});
     await user.save();
 
-    cronMiddleware(req, res, (err) => {
+    cronMiddleware(req, res, () => {
```

Comment:
@FedyuninV 
> @paglias I checked (added manual throw) - it fails one test without interrupting whole bunch of tests.

my comment was referred to what happens if an error happens inside the cronMiddleware code since the error handling callback has been removed in this code. since cronMiddleware does not use promises I don't think the error can be caught by the test suite now that `done` has been removed